File: Data Structures & Algorithms/duplicate-integer/submission-0.py
class Solution:
    def hasDuplicate(self, nums: List[int]) -> bool:
        duplicates = []
        for i in range(len(nums)):
            for j in range(len(duplicates)):
                if (nums[i] == nums[j]):
                    return True
            duplicates.append(nums[i])
        return False

File: Data Structures & Algorithms/rearrange-array-elements-by-sign/submission-0.cpp
class Solution {
public:
    vector<int> rearrangeArray(vector<int>& nums) {
        queue<int> pos = {}; queue<int> neg = {};

        for (int& n: nums) {
            if (n >= 0) {pos.push(n);}
            else {neg.push(n);}
        }

        for (int i = 0; i < nums.size(); ++i) {
            if (i % 2 == 0) {
                nums[i] = pos.front(); pos.pop();
            } else {
                nums[i] = neg.front(); neg.pop();
            }
        }

        return nums;
    }
};